core/objects.go

```diff
@@ -261,7 +261,9 @@ type Authorization struct {
 	Identifier AcmeIdentifier `json:"identifier,omitempty" db:"identifier"`
 
 	// The account key that is authorized for the identifier
-	Key jose.JsonWebKey `json:"key,omitempty" db:"key"`
+	// Key jose.JsonWebKey `json:"key,omitempty" db:"key"`
+	// The registration ID associated with the authorization
+	RegID int `json:"regID,omitempty" db:"regID"`
```

Comment:
For foreign keys I like to consistently name them with the name of the table, so RegistrationID / registrationID.